staging/src/k8s.io/kubelet/pkg/apis/podresources/v1/api.proto

```diff
@@ -82,15 +82,22 @@ message NUMANode {
 
 // DynamicResource contains information about the devices assigned to a container by DRA
 message DynamicResource {
-    string class_name = 1;
+    // tombstone: removed in 1.31 because claims are no longer associated with one class
+    // string class_name = 1;
```

Comment:
Why not just remove it? Is this common practice if a field has crept into a v1 API (even if its usage is hidden behind a feature gate)?

Reply:
Yes, this is common practice. It documents for the future that field number 1 cannot be reused and what it was used for in the past.